go.mod

```
@@ -0,0 +1,5 @@
+module github.com/raymundo/academy-go-q42021
```

Remember that your module is created from your repo tree.

```suggestion
module github.com/rayl17/academy-go-q42021
```